changelogs/current.yaml

```diff
@@ -124,6 +124,10 @@ bug_fixes:
 - area: upstream
   change: |
     Fixed the LOGICAL_DNS and STRICT_DNS clusters to work for IPv6.
+- area: thrift_proxy
```

Comment:
Also, we should probably phrase the release note (and I should rephrase the one for listener stat_prefix) in a way that users know what actually changed, eg, "prometheus metric changed from x to y{labels}".

Reply:
I put it there because your change was also there:

```
- area: listener
  change: |
    Fixed metric tag extraction so that :ref:`stat_prefix <envoy_v3_api_field_config.listener.v3.Listener.stat_prefix>`
    is properly extracted.
- area: upstream
```

No problem moving them but shouldnt they all be together under the same section

Reply:
No problem, will move them both in this PR

Reply:
fixed in 98411acc400a6c93baa0239fb06f88ad8af6e1d3 unless I have an embarrassing typo